Autopilot/Inc/altimeter.hpp

```diff
@@ -35,4 +38,25 @@ class Altimeter{
         virtual void GetResult(AltimeterData_t *Data) = 0; //
 };
 
+class MS5637 : public Altimeter {
+    public:
+        void Init();
+        void Begin_Measuring();
+        void GetResult(AltimeterData_t *Data);
+    private:
+        uint32_t readFromMS5637(uint32_t commandToWrite);
+        void getRawPressureAndTemperature(int64_t *rawPressure, int64_t *rawTemperature);
+        uint32_t getCurrentTime();
+        float getTemperature();
+        float getPressure();
+        float getAltitude();
+        
+        static inline float altitudeMeasured, pressureMeasured, temperatureMeasured;
```

Comment:
inline variables in a c++ 17 feature. At the moment, our build scripts only guarantee support of c++ 11.  This might be why your build is failing.

Reply:
D:

Reply:
@Fberbari Any clue on how to use a solution similar to inline? The only other way I could see us doing this is physically putting it at the top of the cpp file 😨

Reply:
Yeah, that would be fine. Is there something wrong with putting them at the top of the file ? I thought it was pretty standard. For instance in attitudeStateClasses.cpp, in the state machine commit, this section appears at the top:

`/***********************************************************************************************************************
 * Definitions
 **********************************************************************************************************************/

float OutputMixingMode::_channelOut[4];
PMCommands fetchInstructionsMode::_PMInstructions;
SFOutput_t sensorFusionMode::_SFOutput;
PID_Output_t PIDloopMode::_PidOutput;`